### PR TITLE
OIC Connectivity relarted best practice

### PR DESCRIPTION
**Pull request by PeterObertTCE**
Added the first asset to which should be public - already used for several customers 
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by PeterObertTCE** [comment]
I have read the CLA Document and I hereby sign the CLA

File: application-development/application-integration-and-process-automation/oracle-integration-cloud/01-oic-connectivity-agent/Installing-OIC-agent-on-OCI.pdf
Comment (MarcelStraka) [inline application-development/application-integration-and-process-automation/oracle-integration-cloud/01-oic-connectivity-agent/Installing-OIC-agent-on-OCI.pdf:1]: The document should have a title on the first page

File: application-development/application-integration-and-process-automation/oracle-integration-cloud/01-oic-connectivity-agent/README.md
Comment (MarcelStraka) [inline application-development/application-integration-and-process-automation/oracle-integration-cloud/01-oic-connectivity-agent/README.md:1]: This needs to be fixed

**Review by MarcelStraka — CHANGES_REQUESTED** [review]

**Comment by PeterObertTCE** [comment]
going to fix

**Pull request closed without merging**